utils/utils.py

```diff
@@ -0,0 +1,11 @@
+import yaml
+from os import path, getcwd
+from pathlib import Path
+from omegaconf import DictConfig
+
+def read_yaml(path_to_yaml: str):
+    with open(path_to_yaml, 'r') as f:
+        return DictConfig(yaml.load(f.read(), Loader=yaml.FullLoader))
+    
+def get_project_dir():
+    return Path(__file__).parent.parent
```

Comment:
Add some documentation. 
Most importantly, state that get_project_dir is dependent on the location of this file, so people should be careful when moving it around.